pkg/quota/v1/evaluator/core/persistent_volume_claims_test.go

```diff
@@ -195,6 +213,7 @@ func TestPersistentVolumeClaimEvaluatorMatchingResources(t *testing.T) {
 				"persistentvolumeclaims",
 				"gold.storageclass.storage.k8s.io/requests.storage",
 				"gold.storageclass.storage.k8s.io/persistentvolumeclaims",
+				"gold.volumeattributesclass.storage.k8s.io/persistentvolumeclaims",
```

Comment:
Can we also add "gold.volumeattributesclass.storage.k8s.io/requests.storage"?

Reply:
@xing-yang  It is not supported.  discussed in https://github.com/kubernetes/kubernetes/pull/118863#discussion_r1336297758